fe/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Flex,
+  Box,
+  Input,
+  Text,
+  FormLabel,
+  Textarea,
+  Button,
+} from '@chakra-ui/react';
+import {
+  Formik,
+  Form,
+} from 'formik';
+
+interface MyFormValues {
```

Comment:
Name `MyFormValues` doesn't say much about the purpose and usage of this interface. Something like `ProductCreateForm` or `ProductFormValues` sounds better.